inputFiles/poromechanics/PoroDelftEggWellbore_base.xml

```diff
@@ -0,0 +1,232 @@
+<Problem>
+<!-- SPHINX_WELLBORE_TABLE -->
+<Functions>
+  <TableFunction 
+    name="timeFunction"
+    inputVarNames="{time}"
+    coordinates="{0.0, 0.1, 1e6}"
+    values="{0.0, 1.0, 1.0}" 
+  />
+</Functions>
+<!-- SPHINX_WELLBORE_TABLE_END -->
```

Comment:
Do you need these tags?  Do you plan to document this file with Sphinx?

Reply:
Not for now and remove these tags